doc/man3/EVP_DigestInit.pod

```diff
@@ -625,9 +625,9 @@ implementations of digests to be specified.
 If digest contexts are not cleaned up after use,
 memory leaks will occur.
 
-EVP_MD_CTX_name(), EVP_MD_CTX_size(), EVP_MD_CTX_block_size(),
-EVP_MD_CTX_type(), EVP_get_digestbynid() and EVP_get_digestbyobj() are defined
-as macros.
+EVP_MD_CTX_get0_name(), EVP_MD_CTX_get_size(), EVP_MD_CTX_get_block_size(),
+EVP_MD_CTX_get_type(), EVP_get_digestbynid() and EVP_get_digestbyobj() are
+defined as macros.
```

Comment:
Things went a little too automatically, eh?  Considering that you added documentation on the old names further down, you can remove this paragraph.

Reply:
But these calls are defined as macros. IMO this is correct and should be kept.

Reply:
1.  No, EVP_MD_CTX_get0_name(), EVP_MD_CTX_get_size(), EVP_MD_CTX_get_block_size() and EVP_MD_CTX_get_type() are not defined as macros.
2.  You are saying *the same thing* in the paragraph that starts at line 721, i.e. this paragraph has become redundant.

Reply:
Richard, please look into the header. These calls _are_ defined as macros. And the paragraph in the HISTORY section is about the old names.

Reply:
[ahem]

Ok, blushing here...  I kept not seeing `CTX`.  Weird things that a brain can do at times.

So, [ahem] neeeeever mind!